manual_grpc.py

```diff
@@ -0,0 +1,29 @@
+import grpc
```

Comment:
If it is really necessary to include this file in the PR, let's put it in some directory to keep things ordered, otherwise just delete.